.rubocop_styleguide.yml

```diff
@@ -5,8 +5,8 @@
 # rubocop locally, the default configuration file `.rubocop.yml` loads
 # our "todo lists" to ignore all current violations.
 AllCops:
-  TargetRubyVersion: 2.2
```

Comment:
There is probably a conflict here now since we updated Rubocop in master.

Reply:
I think the Rails 3 and Rails 4 rubocop configs are not compatible at all...